src/Microsoft.Azure.EventHubs.Processor/Microsoft.Azure.EventHubs.Processor.csproj

```diff
@@ -5,7 +5,7 @@
     <AssemblyTitle>Microsoft.Azure.EventHubs.Processor</AssemblyTitle>
     <VersionPrefix>1.0.3</VersionPrefix>
     <Authors>Microsoft</Authors>
-    <TargetFrameworks>net451;netstandard1.3;uap10.0;</TargetFrameworks>
+    <TargetFrameworks>net451;netstandard2.0;uap10.0;</TargetFrameworks>
```

Comment:
>net451 [](start = 22, length = 6)

Do you want to move this to 461 as well? [https://github.com/dotnet/standard/blob/master/docs/versions.md](https://github.com/dotnet/standard/blob/master/docs/versions.md)

Reply:
Done